frontend/app/src/components/ToolbarActions/ToolbarActions.tsx

```diff
@@ -65,7 +65,7 @@ function ToolbarActions({
   hostToolbarItems,
 }: ToolbarActionsProps): ReactElement {
   return (
-    <div data-testid="stToolbarActions">
+    <div data-testid="stToolbarActions" style={{ display: "flex" }}>
```

Comment:
nit: I think it would be better to actually add a `StyledToolbarActions` in styled-components with this style. And potentially, it would be also good to add:

```
flex-direction: row;
align-items: center;
```

since these two options were also part of the previous wrapping container (`stToolbar`). But not sure if they are necessary.